ledger/participant-state/kvutils/src/test/suite/scala/com/daml/ledger/participant/state/kvutils/committer/transaction/TransactionCommitterSpec.scala

```diff
@@ -696,7 +696,7 @@ class TransactionCommitterSpec extends AnyWordSpec with Matchers with MockitoSug
       rejectionReason should startWith("DuplicateKeys")
     }
 
-    "return DuplicateKeys when a create in a rollback conflicts with a global key" in {
+    "return InconsistentKeys when a create in a rollback conflicts with a global key" in {
```

Comment:
Please verify that this is what you expect. I think the new behavior is correct:
The global key was not active during submission but it is active now so we get `InconsistentKeys`.

The new code makes this very explicit: `Transaction.keyInputs` checks that the transaction has no internal conflicts which should already have been an error during submission. Anything else is an inconsistency.

Reply:
just to close the loop on this: I reverted to the previous behavior now.